exporter/opentelemetry-exporter-otlp-proto-grpc/tests/test_otlp_trace_exporter.py

```diff
@@ -299,15 +316,35 @@ def test_otlp_exporter_endpoint(self, mock_secure, mock_insecure):
             (
                 "http://localhost:4317",
                 None,
-                mock_insecure,
+                mock_secure,  # Default secure
```

Comment:
This is the case I was referring above. If insecure is not set and scheme is `http` then shouldn't it be insecure_channel?